### ISSUE TITLE
⚠️ Read this about TS parsing errors before submitting an issue ⚠️

### ISSUE THREAD

**Issue by baseballyama** [sveltejs/eslint-plugin-svelte#1111]
There have been reports of the error "Parsing error: Unexpected token {", which occurs due to missing configurations in `eslint.config.js`.

Please refer to the documentation and ensure your settings are correct. If the error persists despite correct configuration, feel free to open an issue.

[[Documentation](https://github.com/sveltejs/eslint-plugin-svelte?tab=readme-ov-file#typescript-project)](https://github.com/sveltejs/eslint-plugin-svelte?tab=readme-ov-file#typescript-project)

**Comment by AlbertMarashi** [comment]
If it helps anyone:
```ts
    {
        files: ["**/*.svelte", "**/*.svelte.ts"],
        languageOptions: {
            parserOptions: {
                projectService: true,
                extraFileExtensions: [".svelte", ".svelte.ts"],
                parser: ts.parser,
                svelteFeatures: {
                    experimentalGenerics: true,
                },
                svelteConfig,
            },
        },
    },
```

**Comment by Lippiece** [comment]
I'm still getting this with config from README (without `svelteConfig`).

It happens when I have `on` inline handlers that are multiline.

**Comment by Lippiece** [comment]
> It happens when I have `on` inline handlers that are multiline.

Scratch that, it happens with TS inside handlers.

**Comment by baseballyama** [comment]
@Lippiece I can not reproduce this. Can you create a new issue with reproduction?

**Comment by Lippiece** [comment]
> [@Lippiece](https://github.com/Lippiece) I can not reproduce this. Can you create a new issue with reproduction?

Thanks for checking this out. I couldn't do it. The same config works if eslint is installed locally, but not globally. Maybe someone else will have an idea.
